### DISCUSSION TITLE
Livewire post 404

### DISCUSSION THREAD

**Discussion by nicholasas1** [livewire/livewire#4862] [1. Help]
please help me, i put livewire in nginx subfolder, and when i run it, i get 404 error like this picture.
![image](https://user-images.githubusercontent.com/92376756/161893046-e8541ab0-9d4a-4a88-ad0e-948cc4408024.png)

I've tried changing .env and config/livewire.php
![image](https://user-images.githubusercontent.com/92376756/161893109-af88e14a-c302-4860-a8f4-9080beb6b563.png)
![image](https://user-images.githubusercontent.com/92376756/161893168-445f61f0-4f74-442d-b45e-de1981d2e06f.png)

can anyone help

**Comment by tomphilpotts** [comment]
Were you able to resolve this? I am having the same issue
